commands-yml/commands/device/network/gsm-call.yml

```diff
@@ -0,0 +1,59 @@
+---
+name: GSM Call
+short_description: Make GSM call (Emulator only)
+
+example_usage:
+  javascript_wd:
+    |
+      await driver.gsmCall('555-123-4567', 'Phone');
+  php:
+    |
+      // TODO
+  csharp:
+    |
+      // TODO
+
+client_docs:
+  javascript_wd: "https://github.com/admc/wd/blob/master/lib/commands.js#L3183"
+  php: "https://github.com/appium/php-client/" # TODO
+  csharp: "https://github.com/appium/appium-dotnet-driver/" # TODO
+
+# Driver support by platform
+driver_support:
+  ios:
+    xcuitest: false
+    uiautomation: false
+  android:
+    uiautomator2: false
```

Comment:
Does this not just fall through to the android driver?

Reply:
I don't know. @SrinivasanTarget you were saying this doesn't work on UiAutomator2?

Reply:
Yes it wont work in UIAutomator2 mode presently. Have already tested it and updated @vrunoa on this.